stack.yaml

```diff
@@ -0,0 +1,12 @@
+resolver: lts-6.27
+
+packages:
+- "."
+- location:
+    git: https://github.com/typeable/dom-parser
+    commit: 0a8ba3a6b5f5ee063d235e287997db7cf12e79ee
```

Comment:
You said you will merge PR in dom-parser, so I dont touch it. When you merge your PR I will upload new version.